src/components/OSCALControlProse.js

```diff
@@ -469,13 +469,10 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
         props.parameters
       );
     });
-
   return (
     <OSCALStatementEditing container spacing={2}>
       <Grid item xs={11}>
-        <StyledTooltip
-          title={statementByComponentDescription ?? props.componentId}
-        >
+        <StyledTooltip title={statementByComponentDescription ?? ""}>
```

Comment:
What's the justification for this change?

Reply:
Actually the commit afterwards makes this useless. I will remove this change.